src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -131,6 +131,16 @@ public static String sanitizeForHtml(String str) {
     }
 
     /**
+     * Escapes HTML tag safely. This function can be applied multiple times.
+     */
+    public static String sanitizeForHtmlTag(String str) {
+        if (str == null) {
+            return null;
+        }
+        return str.replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;");
```

Comment:
I tried running this with only '<' and '>' sanitized ('&' is left intact) :
```java
String text = "<p>This &lt;/p&gt; is sanitized version of &lt;/p&gt;</p>";
Node node = HtmlHelper.getNodeFromString(text);
System.out.println(getNodeContent(node, "", true));
```
I got this:
```html
<p>
  This &lt;/p&gt; is sanitized version of &lt;/p&gt;
</p>
```
Did I miss anything?